### ISSUE TITLE
[ENH] Implement cross-section classification of steel sections following eurocode 3

### ISSUE THREAD

**Issue by nc-hsu** [domthom21/eurocodedesign#20]
**Is your feature request related to a problem? Please describe.**
The class of the cross-section is an important property that influences the procedures for the verification of the resistance of the steel cross-section and the member. This feature has not yet been implemented in this module, and is restricting the further development of the strength verification procedures.

**Describe the solution you'd like**
A function or class method that takes four arguments (a steel section (e.g. `SteelSection`), a steel material (e.g. `BasicStructuralSteel`), and the design axial force and bending moment acting on the cross-section) and return an integer from 1-4 representing the four possible cross-section classes.

**Comment by domthom21** [comment]
Thinks we need to to

- [ ] Do not write commits which contain tests as __main__ method. This should be done in a UnitTest, which can be run seperatly
- [ ] We need documentation for the BasicStructuralSteel properties fy fu, fy_thin, fy thick and so on. I suggest we rename it to f_y_thick or f_y__thick
- [ ] We should define the STEEL_TYPES as a frozen data class in structural steel
- [ ] rename sectionclassification to crosssectionclassification
- [ ] rename get_epsilon to calc_epsilon as this function does something and move to __init__py of ec3
- [ ] add documentation
- [ ] in the file doc we should note the standard and year of the standard
- [ ] add doc which calculations are currently supported and which need further support
- [ ] add units

I think the Cross Section Classification is already huge work and we should work together on this branch.

**Comment by nc-hsu** [comment]
I agree with all of the enhancements that you propose above - however I don't fully understand the first point. You will need to explain to me what it is you mean exactly.

**Comment by domthom21** [comment]
For example, the sectionclass.py in feature-20 has currently a `__main__` method at the end of the file for quick tests.
Changes to this function were in multiple commits. In general these king of quick tests should not be in commits. 
However, you programmed the whole file only for you first and you gave it spontaneously to me.

**Comment by domthom21** [comment]
We should work on some of the issues and start a pull request, because otherwise the changes from branch develop get too large
